transactions/src/main/java/org/ehcache/transactions/xa/internal/SoftLockValueCombinedSerializer.java

```diff
@@ -30,7 +30,7 @@
 class SoftLockValueCombinedSerializer<T> implements Serializer<SoftLock<T>> {
 
   private final AtomicReference<? extends Serializer<SoftLock<T>>> softLockSerializerRef;
-  private final Serializer<T> valueSerializer;
+  final Serializer<T> valueSerializer;
```

Comment:
Can be `protected` no?

Reply:
What would be the point? It would become _more_ accessible.

Reply:
Because we test that `init` is called now. But an integration test will prove that **it is** working. That no other issue is in the way.

Reply:
True... It is just my feeling. When something can be used by a parent class, I don't like to force the parent to be in the same package. But then you will say that the class is already in default scope... Which I missed... So keep it default scope :-)